src/modules/media/const/index.tsx

```diff
@@ -8,6 +8,13 @@ import { Icon } from '@shared/components';
 
 import { DynamicActionMenuProps } from '../components/DynamicActionMenu';
 
+/**
+ * Render media
+ */
+
+export const flowplayerFormats: string[] = ['mp4', 'mp3'];
+export const imageFormats: string[] = ['png'];
```

Comment:
https://flowplayer.com/resources/guides/players/supportedformats
```suggestion
export const flowplayerFormats: string[] = ['mp4', 'mp3', 'm4a', 'ogv', 'ogg', 'webm', 'm3u8'];
export const imageFormats: string[] = ['png', 'jpg', 'jpeg', 'gif'];
```